packages/zwave-js/src/lib/commandclass/ThermostatFanStateCC.ts

```diff
@@ -0,0 +1,174 @@
+import {
+	CommandClasses,
+	Maybe,
+	MessageOrCCLogEntry,
+	MessageRecord,
+	validatePayload,
+	ValueMetadata,
+} from "@zwave-js/core";
+import { getEnumMemberName, pick } from "@zwave-js/shared";
+import type { Driver } from "../driver/Driver";
+import { MessagePriority } from "../message/Constants";
+import {
+	CCAPI,
+	PollValueImplementation,
+	POLL_VALUE,
+	throwUnsupportedProperty,
+} from "./API";
+import {
+	API,
+	CCCommand,
+	ccValue,
+	ccValueMetadata,
+	CommandClass,
+	commandClass,
+	CommandClassDeserializationOptions,
+	expectedCCResponse,
+	implementedVersion,
+} from "./CommandClass";
+
+// All the supported commands
+export enum ThermostatFanStateCommand {
+	Get = 0x02,
+	Report = 0x03,
+}
+
+export enum ThermostatFanState {
+	"Idle / off" = 0x00,
+	"Running / running low" = 0x01,
+	"Running high" = 0x02,
+	"Running medium" = 0x03,
+	"Circulation mode" = 0x04,
+	"Humidity circulation mode" = 0x05,
+	"Right - left circulation mode" = 0x06,
+	"Up - down circulation mode" = 0x07,
+	"Quiet circulation mode" = 0x08,
+}
+
+@API(CommandClasses["Thermostat Fan State"])
+export class ThermostatFanStateCCAPI extends CCAPI {
+	public supportsCommand(cmd: ThermostatFanStateCommand): Maybe<boolean> {
+		switch (cmd) {
+			case ThermostatFanStateCommand.Get:
+				return this.isSinglecast();
+		}
+		return super.supportsCommand(cmd);
+	}
+
+	protected [POLL_VALUE]: PollValueImplementation = async ({
+		property,
+	}): Promise<unknown> => {
+		switch (property) {
+			case "state":
+				return (await this.get())?.[property];
+
+			default:
+				throwUnsupportedProperty(this.ccId, property);
+		}
+	};
+
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+	public async get() {
+		this.assertSupportsCommand(
+			ThermostatFanStateCommand,
+			ThermostatFanStateCommand.Get,
+		);
+
+		const cc = new ThermostatFanStateCCGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+		});
+		const response = await this.driver.sendCommand<ThermostatFanStateCCReport>(
+			cc,
+			this.commandOptions,
+		);
+		if (response) {
+			return pick(response, ["state"]);
+		}
```

Comment:
I think I'd replace this with `return response?.state`.
If the standard ever adds a new property, we can handle it by returning an object for higher versions.